test/test_monitor.py

```diff
@@ -79,6 +80,17 @@ def test_cleanup_executors_on_client_close(self):
         for executor in executors:
             wait_until(lambda: executor._stopped, f"closed executor: {executor._name}", timeout=5)
 
+    @unittest.skipIf(sys.version_info[:2] >= (3, 12), reason="Python version must be (>=3.12)")
+    def test_no_thread_start_runtime_err_on_shutdown(self):
+        """Test we silence noisy runtime errors fired when the MongoClient spawns a new thread
+        on process shutdown."""
+        command = ["python", "-c", "'from pymongo import MongoClient; c = MongoClient()'"]
+        completed_process: subprocess.CompletedProcess = subprocess.run(
+            " ".join(command), shell=True, capture_output=True
```

Comment:
No need to `join` on command since run() accepts a list.

Reply:
I've been attempting to capture the output, and `shell=True` has been the main way for me to capture that output. It seems to not capture all of the shell output when done without the `shell=True`. 

I'm not sure why this is the case and haven't found much good material explaining why, but since these tests are scoped and explicitly defined, I'd argue it's fine to keep this way. The only difference I know is that `shell=True` uses whatever native `sh` the underlying system uses, environment variables and all.

If anyone does know why it's not capturing the output in either stderr/stdout when using `shell=False` that would be great.

Reply:
Oh I see. Thanks for explaining.